### ISSUE TITLE
Unable to resolve symbol: blue-cue

### ISSUE THREAD

**Issue by dandaka** [brunchboy/afterglow#4]
Trying to follow your tutorial. Everything works fine until command

```
afterglow.examples=> (show/add-effect! :color blue-cue)
CompilerException java.lang.RuntimeException: Unable to resolve symbol: blue-cue in this context, compiling:(/private/var/folders/m7/qj5nmqfx353fbx1scql8hkqw0000gn/T/form-init105587552260443348.clj:1:1)
```

Is there anything I'm missing? Or is it a bug?

I am super new to Clojure.

**Comment by dandaka** [comment]
This is my error.log:

```
15-Nov-06 18:28:57 dandaka-mbp.local INFO [afterglow.core] - Loading init-file my-show.clj
15-Nov-06 18:28:57 dandaka-mbp.local ERROR [afterglow.core] - Problem loading init-file my-show.clj
           afterglow.core.main
                           ...
          afterglow.core/-main     core.clj:  279
       afterglow.core/-main/fn     core.clj:  282
                           ...
                my-show/eval42  my-show.clj:    1
my-show/eval42/loading--auto--  my-show.clj:    1
                           ...
          clojure.core/require     core.clj: 5832
            clojure.core/apply     core.clj:  632
                           ...
        clojure.core/load-libs     core.clj: 5749
            clojure.core/apply     core.clj:  632
                           ...
         clojure.core/load-lib     core.clj: 5730
            clojure.core/apply     core.clj:  632
                           ...
            clojure.core/refer     core.clj: 4097
           java.lang.IllegalAccessError: color-create-color does not exist
clojure.lang.Compiler$CompilerException: java.lang.IllegalAccessError: color-create-color does not exist, compiling:(/Users/dandaka/projects/lights/my-show.clj:1:1)
```

**Comment by brunchboy** [comment]
So sorry! I’m almost certain this is due to me changing lots of the underlying code since the last time I tried walking through the examples, and forgetting to update something… Yes indeed, I got rid of the `blue-cue` variable in favor of a new `fiat-lux` function and forgot to update the examples. I am fixing that now, thanks so much for pointing it out!

Can you try calling `(fiat-lux)` instead of using the two expressions that involved `blue-cue` in the current walkthrough, and see if that gets you going again? The error log you reported about `my-show.clj` appears to be a second issue, and I will look at that next, but want to post this response first.

If you would like more rapid help working through the examples (or indeed about anything) feel free to jump on the Gitter chat: https://gitter.im/brunchboy/afterglow

**Comment by brunchboy** [comment]
All right! That error log seems to have been from another run where you were using `java -jar` to run Afterglow outside of Leiningen with the skeletal `my-show.clj` example. I found a mistake in the `:refer` list for `com.evocomputing.colors` and fixed that as well: `color-create-color` should have just been `create-color`, it got munged when I was deleting a reference to `color-name` which used to come before it.

I’ve pushed the corrections to the README, along with some others I found while checking the rest of the walk-through, and will close the issue. Thanks again for reporting this, and I am sorry I set traps for you, it wasn’t on purpose, I promise! :blush: 

Diving into this while being new to Clojure is going to be a challenge, I salute you for experimenting with it, and will be very happy to help you figure out any further issues that come up!
